java/ingest/ingest-core/src/main/java/sleeper/ingest/job/status/IngestJobStatusType.java

```diff
@@ -21,6 +21,9 @@
 
 import java.util.stream.Stream;
 
+/**
+ * An enum for all the states an ingest job can be in.
```

Comment:
We can omit "An enum" here. See the Oracle style guide, "Class/interface/field descriptions can omit the subject and simply state the object": https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html